test/e2e/query_test.go

```diff
@@ -458,6 +458,7 @@ func labelValues(t *testing.T, ctx context.Context, addr, label string, start, e
 	}))
 }
 
+//nolint:unparam
```

Comment:
Why this? Can we just fix this function? (:

Reply:
This one was added by me in my original pr. The error is  

![Screenshot from 2020-09-15 17-57-38](https://user-images.githubusercontent.com/25150124/93269441-f8c6b980-f77c-11ea-9e22-1491777ae3f2.png)

I just use the same query in E2E and I think it is okay.

Reply:
yep leaving it as is.